tools/perf-test-processing/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@fluid-tools/perf-test-processing",
+  "version": "1.0.0",
```

Comment:
Should this version be the same as `">=2.0.0-internal.1.0.0 <2.0.0-internal.2.0.0"`? Why is it 1.0.0

Reply:
I don't think it's necessary, this is an internal tool that we'll only use in the pipelines, right now it doesn't even need to be published to do its job. If it did, its versioning can be independent from the release groups, similar to the build-tools and similar packages.